GNUmakefile

```diff
@@ -312,13 +312,12 @@ e2e-test: dev ## Run the Nomad e2e test suite
 .PHONY: integration-test
 integration-test: dev ## Run Nomad integration tests
 	@echo "==> Running Nomad integration test suites:"
-	go test \
-		$(if $(ENABLE_RACE),-race) $(if $(VERBOSE),-v) \
```

Comment:
Previously when using `TestAgent` there was no hope of using `-race` so I had just removed it. Since switching to running a real agent which gets compiled beforehand... I think it should be fine adding it back? Though it'd really only be confirming the test case code has no races. But it shouldn't hurt.